doc/compodoc_sources/how-to-guides/create-new-entity-type.md

```diff
@@ -1,21 +1,67 @@
+Entity types specify a data structure for records in the case management system,
+defining both the data object as well as some details how the records are displayed in the UI to users.
+There may for example be User, Child or School classes representing the types of objects the users can view in lists and detail forms and interact with.
 
-The [EntityModule](../../modules/EntityModule.html) specifies the base class and generic features, specific entity types can extend upon it.
-For an example, have a look at the [User](../../classes/User.html#source) class that extends `Entity`.
+To adapt the platform to a certain use case, you usually want to adapt entity types or create new ones.
+This defines what fields users can see and edit in forms, among other things.
+
+There are two approaches to define entity types:
+- at runtime in the config file
+- at build time in the code
+
+Unless you have special requirements and write further code specifically for your entity type, 
+you usually should rely on the more flexible and easy approach defining them in the config. 
+
+
+## Defining a custom Entity type only through config
+For general details about the configuration system, have a look at [Concepts / Configuration](../concepts/configuration.html).
+The configuration allows you to customize an Aam Digital instance at runtime without changes to the code base.
+A key part of that is extending existing entity types and defining additional ones.
+
+To configure an entity type, edit the config document in your database (`_id: "Config:CONFIG_ENTITY"`)
+and add a section with id `"entity:MyEntityType"`.
+"entity:" serves as a prefix for any config section that updates or creates an entity type, follow by the name of the type.
+
+Within that section, you can define attributes (i.e. properties = form fields)
+as well as some general ("static") details of the entity type.
+All options are described here: [Concepts / Configuration -> Entity](../concepts/configuration.html).
 
+For example, an entry in the config file for a new entity type could look like this:
+```
+"entity:MyEntity": {
+    "label": "My Entity", // used for titles in the UI
+    "labelPlural": "My Entities",
+    "toStringAttributes": ["firstname", "lastname"], // used when an entity is displayed as a reference/link
+    "attributes": [
+        {"name": "name", "schema": { "dataType": "string", "label": "name" } },
+        {"name": "remarks", "schema": { "dataType": "string", "label": "Further remarks", "editComponent": "EditLongText" } },
+    ]
+}
+```
+
+The [config-fix.ts](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/config/config-fix.ts) in the repository is an example and demo configuration from which you can copy parts of configuration definitions also.
```

Comment:
Why would we have this information here? This is very general and also relevant in other cases. Lets keep all the information about the configuration system in one place.

Reply:
As mentioned above I would like to have all information regarding the config system in one place and not mix this in any way with implementation details. So the hint about the example config should only be required in the documentation of the config once.